docker-compose.yml

```diff
@@ -349,6 +349,23 @@ services:
     volumes:
       - kafkaui-data:/etc/kafkaui
 
+  prometheus:
+    image: prom/prometheus:v3.0.0
+    container_name: prometheus
+    ports:
+      - "9091:9091"
```

Comment:
Do we need to expose those?

Reply:
I would like to enter prometheus dashboard, at least for couple days to config properly